### ISSUE TITLE
"OK" button from in-app notification AlertDialog, missing since release 3.6.0

### ISSUE THREAD

**Issue by Tweener** [OneSignal/OneSignal-Android-SDK#284]
Hi,

I've noticed that since the release version 3.6.0 of your SDK, the "OK" button of the AlertDialog, displayed when a notification is received when the app is in foreground, doesn't show up anymore. Therefore, the only way to dismiss this AlertDialog is by pressing the physical back button of the device.
To be sure, I tested the same code with version 3.5.8, it shows up, as expected. (_See the following screenshots._)

Is this a reported bug already?

Cheers.

![onesignal_3 5 8](https://user-images.githubusercontent.com/596985/29569584-37eba28e-8754-11e7-9e62-1b9df53d2fc7.png)
![onesignal_3 6 0](https://user-images.githubusercontent.com/596985/29569583-37ce639a-8754-11e7-9bc7-7ca1dd2e685c.png)

**Comment by Tweener** [comment]
Up!
Any news about this bug?

This is quite critical since it's blocking releases.
It's not acceptable to not have the "OK" button, which mean we have to use version 3.5.8 of OneSignal SDK for now. **But**, on the other hand, to be compatible with version 26 of Android Support Library, we need at least version 3.6.0 of OneSignal SDK.

Thanks!

**Comment by jkasten2** [comment]
@Tweener Sorry for the delay, we will be fixing this in the next release. We recommend using the notification display option as the alert does not support all features. You can `OneSignal.OSInFocusDisplayOption.Notification` with `inFocusDisplaying` on init to use this option. 
Example:
```Java
OneSignal.startInit(this)
   .inFocusDisplaying(OneSignal.OSInFocusDisplayOption.Notification)
   .unsubscribeWhenNotificationsAreDisabled(true)
   .init();
```

**Comment by Tweener** [comment]
Hi @jkasten2, thanks for your reply.
I'll use this fix for now, until the version 3.6.2 is released. Thanks! 

Any idea about the release date?

**Comment by jkasten2** [comment]
@Tweener This is now fixed in the commit above, added a test to prevent a future occurrence of this.

Shooting for a release the beginning of next week as we include a few more fixes and do some additional testing.
